src/applications/combined-debt-portal/debt-letters/const/diary-codes/debtDetailsCardContent.js

```diff
@@ -33,7 +31,7 @@ export const getDebtDetailsCardContent = (debt, dateOfLetter, amountDue) => {
         bodyText: (
           <p>
             Please contact us online through{' '}
-            <a href="https://ask.va.gov">Ask VA</a> or call us at <ContactDMC />{' '}
+            <a href="https://ask.va.gov">Ask VA</a> or call us at <ContactDMC />
```

Comment:
I think we may still need this `{' '}`. it may render as "[...] overseas)to verify[...]" missing the space after "overseas)"

Reply:
Whoops! good catch